tts/tts-adapters/tts-adapter-sapinative/src/main/resources/transform-ssml.xsl

```diff
@@ -54,9 +54,6 @@
   <xsl:template mode="copy" match="token">
     <!-- tokens are unwrapped because they are not SSML1.0-compliant and not SAPI-compliant-->
     <xsl:apply-templates select="node()" mode="copy"/>
-    <xsl:if test="following-sibling::*">
-      <xsl:text> </xsl:text>
-    </xsl:if>
```

Comment:
@NPavie This change seems to have broken a test (`exampleSSML` in `SapiSSMLTest`). It slipped under my radar. Can you have a look?